cmd/istioctl/main.go

```diff
@@ -99,6 +99,7 @@ var (
 			if len(args) != 0 {
 				return fmt.Errorf("create takes no arguments")
 			}
+			c.SilenceUsage = true
```

Comment:
I think this file is now mostly self-contained, so you can change the root command. We can fix later (maybe if we do not use `RunE` it is not a problem).